### PR TITLE
Add Crowdin

### PR DESCRIPTION
**Pull request by DandelionSprout**
As per their settings page in the screencap below, Crowdin now has 2-factor authentication as well.
image

### CODE REVIEW

**Comment by mxxcon** [comment]
Is there docs: link?

**Comment by DandelionSprout** [comment]
None that I'm aware of, especially since Crowdin's knowledge base (judging by its images) seems to be outdated.

File: _data/developer.yml [hunk 1]
```diff
@@ -164,6 +164,12 @@ websites:
       tfa: Yes
       software: Yes
       doc: https://context.io/security-faq
+
```
Comment (Carlgo11) [inline _data/developer.yml]: a lot of whitespace there. Could you remove it?

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/developer.yml (hunk 1 above)
Comment (DandelionSprout) [reply _data/developer.yml]: Sure thing.

**Review by Carlgo11 — APPROVED** [review]

**Review by mxxcon — APPROVED** [review]

**Pull request merged**